Links_recherche-zu-pull-requests-und-github-allgemein.txt

```diff
@@ -15,4 +15,6 @@ https://www.youtube.com/watch?v=hSbJaIdqwKg
 https://www.youtube.com/watch?v=xyTS2yyOWnA
 https://www.youtube.com/watch?v=Nkz7TnhFvWU
 
-https://www.udemy.com/github-tutorial-deutsch/
\ No newline at end of file
+https://www.udemy.com/github-tutorial-deutsch/
+
+Using GitHub with Mercurial and TortoiseHG: https://www.ryadel.com/en/using-github-with-mercurial-and-tortoisehg/
```

Comment:
I'm quite new to Mercurial. How does it incorporate with GitHub different from git?
What's your operating system?

Reply:
OS: Windows 10 ;-)
Mercurial ist wie git ein DVCS und glücklicherweise recht kompatibel.
Ich nutze Mercurial mittlerweile seit mehr als 15 Jahren, wahrscheinlich weil sich das damals unter Windows mehr angeboten hat als git. Mit einem Standard-PlugIn zu TortoiseHg spricht dieses auch mit github und konvertiert tatsächlich recht gut zwischen git und Mercurial.

siehe hier einen Vergleich der beiden DVCS:
https://www.atlassian.com/blog/software-teams/mercurial-vs-git-why-mercurial